nrf_802154/zephyr/Kconfig.nrfxlib

```diff
@@ -1,9 +1,18 @@
 #
```

Comment:
I thought that this file depends on platform (Zephyr or NCS) and shall be moved to sdk-nrf repository. Do you keep it here intentionally?

Reply:
For the time being I have put this here, because `Kconfig.nrfxlib` at nrfxlib root needs it. I could move the `nrfxlib/Kconfig.nrfxlib` to `sdk-nrf/nrfxlib/Kconfig` right away, but I wasn't sure if VV wanted this change to happen right away.
@tejlmand @carlescufi Do you think I should move the `nrfxlib/Kconfig.nrfxlib` to sdk-nrf in the scope of this change?